dev-clean/grammar/LLVMLexer.g4

```diff
@@ -649,5 +649,5 @@ LOCAL_ID
 
 LABEL_IDENT
     : ( LETTER | DECIMAL_DIGIT ) ( LETTER | DECIMAL_DIGIT ) * ':'
-    | QUOTED_STRING ':'
+    /* | QUOTED_STRING ':' */
```

Comment:
what's the reason/case that we need to comment this?

Reply:
For some reason when there is a colon in the string, it parses incorrectly. See [this discussion](https://rompfgroup.slack.com/archives/C02DKCSV0AF/p1638457632016900).

Reply:
oh, I might have missed that discussion. This is a weird issue.